Tests/DependencyInjection/JMSSerializerExtensionTest.php

```diff
@@ -674,7 +674,7 @@ public function testIncludeInterfaces()
         self::assertSame($expected, $actual);
     }
 
-    private function getContainerForConfigLoad(array $configs, ?callable $configurator = null)
+    private function getContainerForConfigLoad(array $configs)
```

Comment:
Interesting. Can we also remove second argument from `getContainerForConfig` function, please?